content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -98,6 +102,30 @@ $ dvc remote modify mygdfolder gdrive_client_id <client ID>
 $ dvc remote modify mygdfolder gdrive_client_secret <client secret>
 ```
 
+## Configure the GDrive remote with a service account
+
+A
+[service account](https://cloud.google.com/docs/authentication/getting-started#creating_a_service_account)
+can be used to configure the GDrive remote.
```

Comment:
What's a service account? What are the benefits?

Reply:
`dvc remote modify` already mentions `service account` for Google Cloud. It is the same thing for Google Drive as well.
GCP project can have list of service accounts. Each service account has its own email address and some granted access rights, etc.
Benefit to use service account is to remove dependency on some user's account. Also it is recommended way to do some automated actions, like running tests on Travis.

Reply:
Got it. I also found https://cloud.google.com/iam/docs/understanding-service-accounts. I'll review the GC section and maybe copy some info into this one.